app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> str:
+    command_name, source_file, destination_file, *_ = command.split()
```

Comment:
Split into variable and then check that length is exactly 3
after this you can make unpacking